client/cmddata.c

```diff
@@ -185,6 +188,23 @@ int Cmdaskdemod(const char *Cmd)
   return 0;
 }
 
+//this function strictly converts >1 to 1 and <1 to 0 for each sample in the graphbuffer
+int CmdGetBitStream(const char *Cmd)
+{
+  int i;
+  CmdHpf(Cmd);
+  for (i = 0; i < GraphTraceLen; i++) {
+    if (GraphBuffer[i] >= 1) {
```

Comment:
hey, Am I the bad one here?  no no,    by all means..

 for (i = 0; i < GraphTraceLen; i++) GraphBuffer[i] = ( GraphBuffer[i] >= 1 ) ? 1 : 0;